css-view-transitions-2/Overview.bs

```diff
@@ -386,6 +494,27 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 		:: Null or a [=list=] of strings, initially null.
 	</dl>
 
+## Additions to [=captured element=] struct ## {#additions-to-captured-element-struct}
+
+	<dl>
+		: <dfn for="captured element">class list</dfn>
+		:: a [=/list=] of strings, initially empty.
+	</dl>
+
+## Additions to capture algorithms: ## {#additions-to-capture-algorithms}
+<div algorithm="capture old classes">
+When capturing the old state of the view transition, perform the following step when iterating on |captureElements| given |element| and |capture|:
+
+	1. Set |capture|'s [=captured element/class list=] to the [=computed value=] of |element|'s 'view-transition-class'.
+
+</div>
+
+<div algorithm="capture new classes">
+When capturing the new state of the view transition, perform the following step when iterating on |captureElements| given |element| and |namedElements|:
```

Comment:
I suppose the same thing here for this loop: https://www.w3.org/TR/css-view-transitions-1/#capture-new-state-algorithm:~:text=set%20of%20strings.-,For%20each%20element%20of%20every%20element%20that%20is%20connected%2C%20and%20has%20a%20node%20document%20equal%20to%20to%20document%2C%20in%20paint%20order%3A,-If%20any%20flat.

But I also don't want to explode the number of sub-algorithms in the L1 spec. So if you have any other idea to better link to the exact spot that's good to. Another suggestion is, "Do this after step 4.7 of https://www.w3.org/TR/css-view-transitions-1/#capture-the-new-state". If you like that, I'm happy with the same for above as well.

Reply:
That's monkey-patchy, I don't like that... let's see what Tab has to say.